### PR TITLE
Update the 3.8 example for AI Semantic Cache

### PR DESCRIPTION
**Pull request by cloudjumpercat**
Updating the 3.8 example for AI Semantic Cache since we merged in the fixed schema

### CODE REVIEW

File: examples/ai-semantic-cache/_3.8.x.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
   vectordb:
-    dimensions: 1024
-    distance_metric: cosine
     strategy: redis
-    threshold: 0.1
\ No newline at end of file
+    dimensions: 3072
+    threshold: 0.1
+    distance_metric: cosine
+    redis:
```
Comment (cloudjumpercat) [inline examples/ai-semantic-cache/_3.8.x.yaml:12]: @lena-larionova I'm not sure if I did this right. `redis` is required, but then only `sentinel_nodes.host` and `cluster_nodes.ip` inside of that are required. Does that mean I include those two? I'm not sure how "required" works, like when it defaults to something, I don't think we add it to the example. Rendered schema config: https://deploy-preview-7713--kongdocs.netlify.app/hub/kong-inc/ai-semantic-cache/unreleased/configuration/

Comment (lena-larionova) [reply examples/ai-semantic-cache/_3.8.x.yaml:12]: I ran `./plugins validate_examples --version 3.8.x --plugins ai-semantic-cache  --verbose` on this example and got:
```
Validating examples...
---------------------------------
| Plugin: ai-semantic-cache
| Source: ./examples/ai-semantic-cache/_3.8.x.yaml
| Response Code: 400
| Valid?: ❌
| code: 2
| fields: {"config"=>{"vectordb"=>{"redis"=>"required field missing"}}}
| name: schema violation
| message: schema violation (config.vectordb: {
  redis = "required field missing"
})
---------------------------------
Done!
```
So it's expecting some kind of redis config. It doesn't have to be the redis sentinel options you mentioned, but it does need to be _something_. I'm guessing there's a "one of" entity check associated with this config option.

I tested with the following code example, and this one passed validation:

```
name: ai-semantic-cache
config:
  embeddings:
    model:
      provider: openai
      name: text-embedding-3-large
  vectordb:
    strategy: redis
    dimensions: 3072
    threshold: 0.1
    distance_metric: cosine
    redis:
      host: exampleredis.com
      port: 80
```

Test:
```
Validating examples...
---------------------------------
| Plugin: ai-semantic-cache
| Source: ./examples/ai-semantic-cache/_3.8.x.yaml
| Response Code: 200
| Valid?: ✅
---------------------------------
Done!
```

Comment (lena-larionova) [inline examples/ai-semantic-cache/_3.8.x.yaml:12]:
```suggestion
    redis:
      host: exampleredis.com
      port: 80
```

**Review by lena-larionova — APPROVED** [review]
lgtm once the edit is applied

**Pull request merged**